web/src/overlays.ts

```diff
@@ -115,6 +115,7 @@ export function open_overlay(opts: OverlayOptions): void {
     overlay_util.disable_scrolling();
     opts.$overlay.addClass("show");
     opts.$overlay.attr("aria-hidden", "false");
+    $(document).find(":focus").trigger("blur");
```

Comment:
Here's few things we can improve here:

* Let's move the call above `overlay_util.disable_scrolling();` just to make sure it is called before overlay is displayed.
* We can use `document.activeElement` to avoid the extra `find` query.
* We should check the focused element is not `window` since we don't want to call blur on window and trigger unwanted side effects.

Reply:
Yeah those changes sound good -- we want to be certain that we do this before it can possibly have had focus move to the overlay, and it's more obviously that way a bit earlier.

Reply:
I have made these changes